client/app/components/modals/widget-wizard/widget-wizard.controller.js

```diff
@@ -285,6 +285,9 @@ const widgetWizardController = function WidgetWizardController($scope, $mdDialog
             if ($scope.echartConfig.clear && $scope.echartConfig.previousTemplate && $scope.echartConfig.previousTemplate.type !== $scope.echartConfig.currentTemplate.type) {
                 $scope.echartConfig.clear();
             }
+            if ($scope.echartConfig.isDisposed && !$scope.echartConfig.isDisposed()) {
```

Comment:
Confusing code: `$scope.echartConfig.isDisposed()` returns nothing, so why it is in condition?